pkg/cloudprovider/aws.go

```diff
@@ -20,6 +21,7 @@ import (
 const (
 	AwsAccessKeyId     = "aws-access-key-id"
 	AwsSecretAccessKey = "aws-secret-access-key"
+	AwsS3DefaultRegion = "us-east-1"
```

Comment:
This _const_ block is for the credentials secret data.  Please add in a new _const_ block.